practice-two/src/components/FormValidation/FormValidation.jsx

```diff
@@ -67,10 +88,22 @@ const FormValidation = (props) => {
       setErrors(newErrors);
 
       if (Object.keys(newErrors).length === 0) {
-        onShowForm();
+        const response = await createCustomerService(formData);
+
+        if (response.error) {
+          handleShowForm();
```

Comment:
why need to show form when response error?

Reply:
![image](https://github.com/datdragon777/React-Training/assets/74806826/cf75a132-ebd4-4b06-a92c-61f5adbcd994)
![image](https://github.com/datdragon777/React-Training/assets/74806826/ed3e0f9e-dbef-4dca-be82-8b0a3fb942d2)

Actually, it will close form. In parent component, when I open the form, the isShowForm will change to true
When form is opened, user click Cancel or Create button, the isShowForm will chang to false, it means form will close
In that case, when occuring some errors in server while creating, user click Create button and it will close form and give a Toast message to inform that create customer failed